packages/calypso-color-schemes/bin/build-css.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable import/no-nodejs-modules */
+const { dirname, join } = require( 'path' );
+const { existsSync, mkdirSync, writeFileSync } = require( 'fs' );
+const { renderSync } = require( 'node-sass' );
+
+const INPUT_FILE = join( __dirname, '..', 'src', 'custom-properties.scss' );
+const OUTPUT_FILE = join( __dirname, '..', 'dist', 'custom-properties.css' );
```

Comment:
should we rename this?

Reply:
To `calypso-color-schemes` (src and dest files)?

Reply:
35778f0400a0d53307ca87a57243285deb4da9f9